pkg/controller/core/workload_controller.go

```diff
@@ -467,9 +468,15 @@ func (r *WorkloadReconciler) triggerDeactivationOrBackoffRequeue(ctx context.Con
 		Steps:    int(requeuingCount),
 	}
 	var waitDuration time.Duration
+	maxWaitDuration := time.Duration(r.waitForPodsReady.requeuingBackoffMaxSeconds) * time.Second
```

Comment:
We could record this on a field of the controller to avoid re-computation.

Reply:
I've modified this to record the value in RequeueState.

Reply:
Adding a dedicated API field (which is close to impossible do remove) just for this is a bit too much in my opinion, I'd go with a "magic value" for `status.RequeueState.Count` (say `-1` , `maxInt` ), indicating that the maximum value should be used.

@alculquicondor @mimowo WDYT?

Reply:
Sorry for not clarifying enough, I was thinking about storing it only in memory, not API, as an internal field of the reconciler when instantiating it

Reply:
Done. I've added the value to waitForPodsReadyConfig in WorkloadReconciler